lib/components/user/monitored-trip/trip-basics-pane.tsx

```diff
@@ -66,7 +67,7 @@ const AvailableDays = styled(FieldSet)`
     }
   }
   & > span {
-    border: 1px solid #ccc;
+    border: 1px solid ${grey[100]};
```

Comment:
Reuse `BIORDER_COLOR` from `trip-summary-pane` (2x).

Reply:
I'd be interested to hear how @miles-grant-ibigroup and @josh-willis-arcadis feel about this. I see what you're saying, but I'm hesitant to create any sort of global variable called BORDER_COLOR since we don't use this for that many borders. Maybe exporting from `trip-summary-pane` makes it less confusing? I don't feel super strongly about this!

Reply:
Yeah I'm not huge about BORDER_COLOR global, but maybe we could export it from some utils file?

Reply:
If we are not using it with more borders I feel like it is fine to not make a global variable out of it. I like the idea to export it so it can be used as needed.

Reply:
Okay! I exported `BORDER_COLOR` from `trip-summary-pane` and used in in the two places Binh pointed out.